index.html

```diff
@@ -68,5 +68,152 @@ <h2 class="intro2">I’am a Software Developer</h2>
         </li>
       </ul>
     </section>
+    <section class="section-two">
```

Comment:
Excelent work adding the finals steps on this project 💯 

- [ ] Kindly add a `max-width: 270px` to your `.description` class, so in this way, the description will not overlap with your first section 🥇 

![image](https://user-images.githubusercontent.com/93630700/213740442-9ad594d7-2e28-4791-bad3-5d544b9f60b1.png)